build.gradle

```diff
@@ -134,30 +130,11 @@ applicationDistribution.into('bin') {
     fileMode = 0755
 }
 
-// Import task types
-import com.bmuschko.gradle.docker.tasks.image.*
-
-// Use task types
-task pullBuildDocsImage(type: DockerPullImage) {
-    repository = "pseudomuto/protoc-gen-doc"
-}
-
-configurations {
-    dockerRun {
-        name 'build-docs-container'
-        image 'pseudomuto/protoc-gen-doc'
-        volumes "src/main/docs": "/out", "src/main/proto": "/protos"
-        clean true
-    }
+task buildGrpcGateway(type: Exec) {
+    workingDir = '.'
+    commandLine = './build_grpc_gateway.sh'
 }
 
-task buildDocs() {
```

Comment:
Its original motivation was to provide documentation. It is currently broken for jdk13 #16  as you mentioned. Since swagger allows us to provide documentation I did not see it necessary to have this broken code exist since its functionality is provided by swagger. If we do plan to have multiple ways to provide documentation then it might make more sense to have working code checked in the repo?